doc_src/getting_started.md

```diff
@@ -35,5 +35,5 @@
     - Take a look around and play around with it.
     - Check out documentations and see what you can do with antikythera and gears.
         - [Guide for instance administrators](./instance_administrators/) (TBD)
-        - [Guide for gear developers (i.e. developers of web services)](./gear_developers/)
+        - [Guide for gear developers (i.e. developers of web services)](./gear_developers/README.md)
```

Comment:
I think that whenever possible, the link to the document should be a link to `hexdocs.pm`.
```suggestion
        - [Guide for gear developers (i.e. developers of web services)](https://hexdocs.pm/antikythera/gear_developers.html)
```

Reply:
In my opinion, a link in a Markdown file should point to a file in the repository.
I expect that the link will be converted properly when generating hex document.

Reply:
It may be useful when adding new documents, but this time it is unnecessary.
Links to the documentation in this repository are already standardized at `hexdocs.pm`. (e.g. 5022b90be93790bd77a90513cd8ff2d81c0b6abd)

Reply:
Thank you for your suggestion.
I applied.
047962fcbc7888702bb3a341b463b7f55c46d1b1